app/main.py

```diff
@@ -2,0 +1,13 @@
+def copy_file(command: str) -> None:
+
+    if (len(command.split()) != 3 and command.split()[0] != "cp"
+            and command.count(" ") != 2):
+        return
+    command_parts = command.split()
```

Comment:
Consider renaming 'command_parts' to something more descriptive like 'file_names' to improve code readability.